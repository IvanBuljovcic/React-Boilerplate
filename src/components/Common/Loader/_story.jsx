import React from 'react';
import { storiesOf } from '@storybook/react';

// - Component
import Loader from './Loader';

const stories = storiesOf('Common', module);


stories.addWithChapters('Loader', {
	chapters: [
		{
			sections: [
				{
					title: 'Default',
					sectionFn: () => <Loader />
				}
			]
		}
	]
})
