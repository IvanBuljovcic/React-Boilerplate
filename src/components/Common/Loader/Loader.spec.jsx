import React from 'react';
import renderer from 'react-test-renderer';

// - Test component
import Loader from './Loader.jsx';

test('<Loader /> renders correctly', () => {
  const tree = renderer.create(<Loader />).toJSON();

  expect(tree).toMatchSnapshot();
});
