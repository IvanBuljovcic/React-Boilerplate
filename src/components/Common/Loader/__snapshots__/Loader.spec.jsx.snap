// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> renders correctly 1`] = `
<div
  className="lds-ring"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
